frontend/scripts/tool/update_collab_source.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Paths to your Cargo.toml files
+REPO_PATH="./AppFlowy-Collab"
```

Comment:
If you're in a fork of AppFlowy, and inside the frontend working directory, then isn't this relative path in tracking for AppFlowy? Would it make more sense to have it one more step back?

Reply:
it's ignored